### DISCUSSION TITLE
rabbitmq-server repository does not seems to be in ppa1.novemberain.com

### DISCUSSION THREAD

**Discussion by maheus** [rabbitmq/rabbitmq-server#8949] [Other] [answered]
### Describe the bug

rabbitmq-server repository does not seems to be in ppa1.novemberain.com

### Reproduction steps

bookworm curl InRelease file retrurn 404
```
 curl -I  https://ppa1.novemberain.com/rabbitmq/rabbitmq-server/deb/debian/dists/bookworm/InRelease
HTTP/2 404 
```

bullseye curl InRelease file retrurn 404
```
curl -I  https://ppa1.novemberain.com/rabbitmq/rabbitmq-server/deb/debian/dists/bullseye/InRelease
HTTP/2 200 
```

### Expected behavior

In your documentation : (https://www.rabbitmq.com/install-debian.html#apt-cloudsmith)
```
## Community mirror of Cloudsmith: RabbitMQ repository
```
and rabbitmq-server bookworm package exists in Cloudsmith  so I suppose the mirror repository did not updated for the bookworm packages.
But I'm not sure you manage the mirror repository

**Comment by michaelklishin** [comment, marked as answer]
`bookworm` was not mirrored due to a typo in `mirror.list`. `bullseye` returns 200 OK for me.

You can use a different distribution name in any case, it's the same RabbitMQ package for all of them (unlike Erlang packages).

**Reply by maheus** [reply]
thanks for your response and for the fix
